### PR TITLE
Ks 2021 08 post category and labels

### PR DESCRIPTION
**Pull request by Rineee**
Finally ... only works with https://github.com/liqd/adhocracy-plus/pull/1394

### CODE REVIEW

File: containers/Ideas/IdeaCreate.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import * as yup from 'yup';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import Icon from 'react-native-vector-icons/SimpleLineIcons';
 import IconSLI from 'react-native-vector-icons/SimpleLineIcons';
+import DropDownPicker from 'react-native-dropdown-picker';
```
Comment (fuzzylogic2000) [inline containers/Ideas/IdeaCreate.js:9]: I guess github actions is right here: should be removed if it's not used.

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! Works well! Just the one tiny thing, we can also do later.

File: containers/Ideas/IdeaCreate.js (hunk 1 above)
Comment (Rineee) [reply containers/Ideas/IdeaCreate.js]: Its removed!

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**